galaxy2cwl/get_cwl_interface.py

```diff
@@ -59,12 +59,25 @@ def process_format1_json(wf_dict):
                 output_name = 'output'
             map_output_to_in_name[step_index + '_' + output_name] = input_name
             wf_inputs[input_name] = input_details
+        elif step_details['type'] == 'parameter_input':
+            input_details = {}
+            input_details['format'] = 'boolean'
```

Comment:
this should check the real format of the input. parameter_input type can be test, boolean, int...

Reply:
You mean format, because the type I read out a bit further? and where can I read it out? Or do I have to guess from the parameter that it is a boolean?

Reply:
if you create a workflow in Galaxy and define a parameter_input that can be integer, float, etc..
This ends up in the tool_state key in the .ga file. Something like:
`tool_state": "{\"parameter_type\": \"float\", \"optional\": false}",`
that needs to be parsed, otherwise here you assume its a boolean

Reply:
got it, will fix it ;)